src/test/scala/com/lunatech/pointingpoker/APISpec.scala

```diff
@@ -8,25 +8,30 @@ import akka.http.scaladsl.model.ws.BinaryMessage
 import akka.http.scaladsl.testkit.{ScalatestRouteTest, WSProbe}
 import com.lunatech.pointingpoker.config.ApiConfig
 import com.typesafe.config.ConfigFactory
-import org.scalatest.{BeforeAndAfterAll, MustMatchers, WordSpec}
 import akka.http.scaladsl.server._
 import akka.util.ByteString
```

Comment:
Just a general note, this is an unused import and there are a few others in this file as well.